server/lib/facebook/facebook_feed_stream.js

```diff
@@ -2,30 +2,23 @@
  * Module streas facebook feed
  **/
 
-import { PassThrough } from 'stream';
+import AbstractFeedStream from './../abstract_feed_stream';
```

Comment:
@asiakaczmar, it's quite [hot topic](https://news.ycombinator.com/item?id=7270436) :bomb: I've added a separate [ticket](https://trello.com/c/DfY7S7XQ/11-ob-11-get-rid-of-long-require-paths) to investigate and solve this problem. Thanks for pointing this out! :+1: